macaroonbakery/discharge.py

```diff
@@ -113,7 +119,7 @@ def discharge(ctx, id, caveat, key, checker, locator):
     party caveats returned by the Checker.
     '''
     caveat_id_prefix = []
-    if caveat is None:
+    if caveat is None or caveat == b'':
```

Comment:
When does this happen? This doesn't match the doc comment description - why do we need to check against the empty string too?

Reply:
my fault